core/shared/src/main/scala/plotly/Trace.scala

```diff
@@ -237,18 +237,24 @@ object Histogram {
 }
 
 @data class Surface(
+          x: Option[Sequence],
+          y: Option[Sequence],
```

Comment:
Turns out that in order to get something like below you need the x and y axes as well. I though you could provide these via the axes in the scene (part of the layout) but you cannot.

![surface](https://user-images.githubusercontent.com/22156776/69426228-0cf08480-0d2d-11ea-9370-b82095817ac8.png)